src/emqx.appup.src

```diff
@@ -1,6 +1,9 @@
-%% -*-: erlang -*-
+%% -*- mode: erlang -*-
 {VSN,
  [
+   {"4.3.2", [
```

Comment:
Probably I need to include all files that use `?tp` macro here...

Reply:
Hi, @k32 I think there is a problem with appup, emqx.app 4.3.2 has not been released yet

Reply:
fixing in this PR: https://github.com/emqx/emqx/pull/4873